src/cli/package.rs

```diff
@@ -131,6 +137,26 @@ impl Packager {
 
                     let wasm = build.run(&node)?;
 
+                    base64::decode(&wasm).ok().and_then(|wasm_binary| {
+                        let context= test_context("HC");
+                        let result = ribosome::run_dna(
+                            &"HC",
+                            context,
+                            wasm_binary,
+                            &ZomeFnCall::new(&"", &"", &"__hdk_get_json_definition", &""),
+                            Some(String::from("{}").into_bytes())
+                        );
+                        println!("RESULT: {:?}", result);
+                        result.ok()
+                    }).and_then(|call_result| {
+                        println!("WASM2JSON: {}", call_result);
+                        serde_json::from_str(&call_result).ok()
+                    }).and_then(|json_from_wasm| {
+                        let parent_filename= util::file_name_string(&node.parent().unwrap().to_path_buf()).unwrap();
```

Comment:
This was an attempt to attach it to the level above but it just changes the root's name ;) still on the level of "code", but with "code" replaced by "blog".. :/